src/components/Dropdown/Dropdown.scss

```diff
@@ -48,18 +60,6 @@
     }
   }
 
-  &:after {
-    width: 24px;
-    height: 24px;
-
-    display: flex;
-    align-items: center;
-
-    content: '\25BE';
-    color: #757575;
-    color: var(--icon-color);
-  }
```

Comment:
removed icon and made it an actual DOM element instead. see comment above for reasoning.